dev/manual_tests/overlay_geometry.dart

```diff
@@ -163,8 +163,9 @@ class OverlayGeometryAppState extends State<OverlayGeometryApp> {
     });
   }
 
-  void handleTapUp(GlobalKey target, Point globalPosition) {
+  void handleTapUp(Key t, Point globalPosition) {
```

Comment:
The original signature would result in a runtime strong mode error.
as
(GlobalKey, Point)
does not match the function signature
(Key, Point)